frontend/packages/dev-console/integration-tests/features/vulnerability/project_vulnerability_ODC-4882

```diff
@@ -0,0 +1,102 @@
+Feature: Image Vulnerability for each Project
+    As a user I would like to see Image Vulnerability for each Project
+
+Background:
+    Given user is in Developer perspecitve
+    And open project namespace
+    And quay container security operator installed
+
+
+@regression
+Scenario: Image Vulnerability pop over in project overview    
+   Given user is in topology of a project
+   When user goes to project tab on sidebar
+   And user sees Status section in overview tab
+   And user sees vulnerabilities with number mentioned in Status section
+   And user clicks on the Image Vulnerabilities
+   Then user sees Image Vulnerability breakdown pop over
+   And user sees Vulnerable Container Images and Fixable Container Images
+   And user closes the pop over 
+
+@regression
+Scenario: Going to Image Manifest Vulnerabilities page from status in project overview    
+   Given user is in project tab
+   And user sees vulnerabilities with number mentioned in Status section
+   When user clicks on the Image Vulnerabilities
+   And user sees Image Vulnerability breakdown pop over
+   And user sees View all link on bottom of pop over
+   And user clicks on the View all link
+   Then user can see Image Manifest Vulnerabilities page
+
+@regression
+Scenario: Going ImageManifestVuln from status in project overview    
+   Given user is in project tab
+   And user sees vulnerabilities with number mentioned in Status section
+   When user clicks on the Image Vulnerabilities
+   And user sees Image Vulnerability breakdown pop over
+   And user sees vulnerabilities with number mentioned in Status section
+   And user clicks on the image name to open that particular ImageManifestVuln
+   Then user can see ImageManifestVuln for the image name
+
+@regression
+Scenario: Going to quay page from status in project overview    
+   Given user is in project tab
+   And user sees vulnerabilities with number mentioned in Status section
+   When user clicks on the Image Vulnerabilities
+   And user sees Image Vulnerability breakdown pop over
+   And user sees Fixable Container Images section
+   And user sees fixable link for namespaces
+   And user clicks on the fixable link 
+   Then user can see the image open in quay.io
+
+@regression
+Scenario: Vulnerability tab in developer console    
+   Given user is in topology of a project
+   When user goes to project tab on sidebar
+   And user sees vulnerability tab
+   And user clicks on the vulnerability tab
+   Then user sees Image Manifest vulnerabilities
+   And user sees filter with name option selected
+   And user can see image name, namespace, highest severity, pods,fixable and manifests
```

Comment:
If a particular project is selected then the user can not see the namespace column.